core/lib/web3_decl/src/error.rs

```diff
@@ -39,10 +39,12 @@ pub enum Web3Error {
     LogsLimitExceeded(usize, u32, u32),
     #[error("invalid filter: if blockHash is supplied fromBlock and toBlock must not be")]
     InvalidFilterBlockHash,
-    #[error("Not implemented")]
-    NotImplemented,
-
-    #[error("Tree API is not available")]
+    /// Weaker form of a "method not found" error; the method implementation is technically present,
+    /// but the node configuration prevents the method from functioning.
+    #[error("Method not implemented")]
```

Comment:
Will this change be breaking if someone depends on the exact error message?

Reply:
Yep, I wanted to comment on this as well. Do we know whether anyone depends on exact error messages returned by the RPC server? Obviously, changing the message isn't critical, so if this is a concern, I can change it back.

Reply:
Well, I think generally we would expect people _not_ to depend on the error message (especially given that there are error codes), so probably it's fine to do it this time.